### ISSUE TITLE
createFontStack returns incorrect values for fonts with the same metrics

### ISSUE THREAD

**Issue by KelvinOm** [seek-oss/capsize#136]
We use `appleSystem` and `BlinkMacSystemFont` metrics to create a font stack. 

```
createFontStack([appleSystem, BlinkMacSystemFont])
```

Calling this method returns this
```
@font-face {
  font-family: "-apple-system Fallback";
  src: local('BlinkMacSystemFont');
  ascent-override: 95.2148%;
  descent-override: 20.5078%;
}
```

Both fonts have the same meters([here](https://github.com/seek-oss/capsize/blob/master/packages/metrics/src/entireMetricsCollection.json#L2-L11) and [here](https://github.com/seek-oss/capsize/blob/master/packages/metrics/src/entireMetricsCollection.json#L24-L34)) and the return value should not have `ascent-override` and `descent-override` styles. The calculation of these parameters is performed by [this code](https://github.com/seek-oss/capsize/blob/master/packages/core/src/createFontStack.ts#L22-L38). I think that there should be a check of metrics and if the values match, then we should not calculate `ascent-override` and `descent-override` parameters.

**Comment by michaeltaranto** [comment]
Thanks for raising this, let me take a look and loop back.

**Comment by michaeltaranto** [comment]
Fix published in [`@capsizecss/core@3.1.1`](https://github.com/seek-oss/capsize/releases/tag/%40capsizecss%2Fcore%403.1.1)

Thanks again
